### DISCUSSION TITLE
How does refunding orders work? Needs to be done via Paymentprovider?

### DISCUSSION THREAD

**Discussion by nabbl** [unchainedshop/unchained#347] [Q&A] [answered]
see title.

**Comment by pozylon** [comment, marked as answer]
Refunding is out of scope of the current Unchained Features, so it's up to you how you want to build that process and like you said usually you have to refund through the actual payment gateway. If the payment is invoicing, it's some other system or manual anyways. Maybe we can make it possible to trigger "credit notes" in Unchained and then try to do refunding with the payment provider plugins that support it.
